packages/assemble-lite/helper/helpers-helper.js

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
path: "/helper/helpers-helper.js" ? are these helpers for the helper ? :D
Eventually "handlebars-helper" or "hbs-helper" is a better filename.

Reply:
its for handlebars-helpers loading so i followed my pattern ([purpose]-helper.js)